package/verification-template/build.gradle

```diff
@@ -18,7 +18,7 @@ configurations {
 dependencies {
 	core group: "net.runelite", name: "client", version: file("../../runelite.version").text.trim()
 	core "org.projectlombok:lombok:1.18.30"
-	core "org.jetbrains:annotations:23.0.0"
+	core "org.jetbrains:annotations:20.0.0"
```

Comment:
What is this downgraded for?

Reply:
first few attempts to send pull request failed, so I was comparing my code with other's, and found that this version on mine was different so I downgraded it to someone elses

I changed it back to 23.0.0, and it seems fine now